apps/richie/templates/postgresql/svc.yml.j2

```diff
@@ -2,25 +2,28 @@ apiVersion: v1
 kind: Service
 metadata:
   labels:
-    app: postgresql
-    version: "{{ postgresql_tag }}"
+    app: richie
+    service: postgresql
+    version: "{{ richie_postgresql_image_tag }}"
     deployment_stamp: "{{ deployment_stamp }}"
   # name of the service should be database host name in settings
-  name: "{{ postgresql.host }}"
+  name: "{{ richie_postgresql_host }}-{{ deployment_stamp }}"
   namespace: "{{ project_name }}"
 spec:
   ports:
-    - name: {{ postgresql.port }}-tcp
-      port: {{ postgresql.port }}
+    - name: {{ richie_postgresql_port }}-tcp
+      port: {{ richie_postgresql_port }}
       protocol: TCP
-      targetPort: {{ postgresql.port }}
+      targetPort: {{ richie_postgresql_port }}
 # As commented in the ad hoc endpoint, the endpoint name points to this service
 # so that it does not rely on a deployment configuration when the "env_type" is
 # not "development" or "feature" (not yet implemented). In environments other
 # than "development" or "feature", we use a PostgreSQL cluster outside of
 # OpenShift.
-{% if env_type == "development"  %}
+{% if env_type in ("ci", "development", "feature")  %}
```

Comment:
@cchaudier @rouja is there a way to define constants so that "development", "ci", "feature" are defined only once for all?

Reply:
maybe this? https://devops.stackexchange.com/questions/3474/how-to-define-constants-in-ansible

Reply:
What about the following?

```yaml
# group_vars/all/main.yml
trashable_env_types:
  - "ci"
  - "development"
  - "feature"

# apps/richie/templates/postgresql/svc.yml.j2
{% if env_type in trashable_env_types %}
# ...
```

Reply:
And also the answer to this https://devops.stackexchange.com/questions/3474/how-to-define-constants-in-ansible advising to user `defaults/main.yml` to define kind of "constants". The idea is to replace everywhere we wrote "development" by this variable. WDYT?

Reply:
`group_vars/all/main.yml` are our defaults that we can override.

> The idea is to use this constant everywhere we wrote "development".

Ok to use it for service templates (these are the only place where we use this condition).